pyenttec/test.py

```diff
@@ -1,15 +1,48 @@
 """Tests for offline DMX port."""
-from nose.tools import assert_equal, assert_raises
-from . import DMXConnectionOffline, DMXAddressError
+from nose.tools import assert_equal, assert_raises, raises
+from . import DMXConnectionOffline, DMXAddressError, DMXOverflowError
+
+def get_test_port(univ_size=24):
+    return DMXConnectionOffline(univ_size=univ_size)
 
 def test_offline_port():
     univ_size = 24
-    port = DMXConnectionOffline(univ_size=univ_size)
+    port = get_test_port(univ_size)
 
     assert_equal(univ_size, len(port.dmx_frame))
     assert all(chan == 0 for chan in port.dmx_frame)
 
     port.set_channel(0, 1)
     assert_equal(1, port.dmx_frame[0])
 
-    assert_raises(DMXAddressError, port.set_channel, univ_size, 0)
\ No newline at end of file
+    assert_raises(DMXAddressError, port.set_channel, univ_size, 0)
+
+def test_port_item_accessors():
+    univ_size = 24
+    port = get_test_port(univ_size)
+    port[univ_size - 1] = 2
```

Comment:
Very easy here to roll a test that exercises the API harder; in the same LOC count:

```python
for i in range(univ_size):
    port[i] = i
    assert_equal(i, port.dmx_frame[i])
    assert_equal(i, port[i])
```

Reply:
Sure, good point